features/step_definitions/resources.rb

```diff
@@ -221,3 +221,18 @@
     cb[cb_name] = clazz.list(user: _user)
   end
 end
+
+Given /^I remove all #{WORD}(?: in the#{OPT_QUOTED} project) with labels:$/ do | resource_type, namespace, table |
+  ensure_admin_tagged
+  ensure_destructive_tagged
```

Comment:
2 lines to remove, only for cluster resources are needed and I see you have there already